directus/save-data.sh

```diff
@@ -0,0 +1,3 @@
+docker exec -it clic-postgres pg_dump -U directus_user --column-inserts --data-only directus_data > directus/new-dump.sql
+echo '[INFO]: Database dump saved at "directus/new-dump.sql"'
+echo '[INFO]: Please make sure to remove undesired inserts (e.g. in the directus_activity table) before committing it'
```

Comment:
This should also be in the README or in a documentation about development imo.

Reply:
I've updated the print to match the comment in the directus readme.